themes/_core/js/cart.js

```diff
@@ -4,13 +4,13 @@ import prestashop from 'prestashop';
 $(document).ready(() => {
   prestashop.on('cart updated', function (event) {
     var refreshURL = $('.-js-cart').data('refresh-url');
-    var requestData = {};
+    var requestData = {
+      action: 'refresh'
```

Comment:
parameter  `-js-cart` is loaded from https://github.com/PrestaShop/PrestaShop/blob/develop/themes/classic/templates/checkout/_partials/cart-summary.tpl#L1

`<section id="checkout-cart-summary" class="card card-block -js-cart" data-refresh-url="{$urls.pages.cart}?ajax=1">`

there is none `action=refresh` parameter